### ISSUE TITLE
Ignore common tools' pragmas in ``line-too-long`` (``# type: ignore``, ``# noqa: RUF001``, ``pragma: no cover`` etc)

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [pylint-dev/pylint#10172]
### Current problem

Sometimes a pragmas comment increases the line length to the point of triggering Pylint line-too-long warning.

### Desired solution

Pylint should ignore those:

```python
VERY_LONG_VARIABLE_NAME_YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY: float
VERY_LONG_VARIABLE_NAME_YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY = 42424  # type: ignore
VERY_LONG_VARIABLE_NAME_ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ = "test"  # noqa: E501
another_long_variable_name = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"  # noqa: E501, RUF001  # pragma: no cover
another_long_variable_name = "yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy"  # pylint: disable=C901
```

### Additional context

Based on https://github.com/pylint-dev/pylint/issues/8378
